### PR TITLE
feat: terraform to launch EKS cluster

### PR DESCRIPTION
**Pull request by rubenruizdegauna**

### CODE REVIEW

**Comment by rogercoll** [comment]
Other than the comment, lgtm

File: entrypoint.sh [hunk 1]
```diff
@@ -46,6 +46,7 @@ if [ -z "${environmentName}" ]; then
 fi
 if [ ! -f "environments/${environmentName}/caos.auto.tfvars" ]; then
   echo "Rename "environments/${environmentName}/caos.auto.tfvars.dist" to "environments/${environmentName}/caos.auto.tfvars", fill the variables and re-run command"
+  echo "cp environments/single-from-otel-ec2/caos.auto.tfvars.dist environments/single-from-otel-ec2/caos.auto.tfvars"
```
[the remark this answers is not in the document]
Comment (rubenruizdegauna) [reply entrypoint.sh:49]: It is mean to show the rename action just to copy and paste. Changing the `single-from-otel-ec2` to `${environmentName}`

[the remark this answers is not in the document]
Comment (rogercoll) [reply entrypoint.sh]: Cool! Thanks

**Review by rogercoll — APPROVED** [review]
LGTM

**Pull request merged**